eZ/Publish/Core/REST/Server/Controller/Root.php

```diff
@@ -20,13 +20,23 @@
 class Root extends RestController
 {
     /**
+     * @var RootResourceBuilder
+     */
+    private $resourceService;
+
+    public function __construct($resourceService)
```

Comment:
Rename `resourceService` to `rootResourceBuilder` (or something about the builder).